user/reference/xenial.md

```diff
@@ -3,6 +3,8 @@ title: The Xenial Build Environment
 layout: en
 ---
 
+> Xenial LTS Standard is EOL by Canonical, try updating to a newer image. All work in Travis CI over updates to Xenial images is ceased with end of calendar year 2024 and we consider it being deprecated.
```

Comment:
Xenial LTS Standard is EOL by Canonical, try updating to a newer image. All work related to updates for Xenial images on Travis CI will cease by the end of the calendar year 2024, and we will consider it deprecated.